.idea/dictionaries/jean_christophe.xml

```diff
@@ -0,0 +1,9 @@
+<component name="ProjectDictionaryState">
```

Comment:
What is this file used for?

Reply:
.idea files are not compiled. This is the dictionary used on our IDE